unitpackage/__init__.py

```diff
@@ -0,0 +1,51 @@
+import yaml
+
+from unitpackage.entry import Entry
+
+
+def csv_to_package(csvname, yamlname, outdir, fields=None):
```

Comment:
This should be moved in a separate module. (which one?) 
This also seems to be a piece for a CLI.

Reply:
The name is a bit misleading. This is really very specific to our "from figure" use case.

Why do you think that this function is important to have for the general public? Or is this just something that should be an example use case in the documentation?

Otherwise, if you drop the `save()` at the end, it could maybe be `Entry.from_parsed_figure()`?

Reply:
This fits indeed much better in the documentation.